jenkins-plugin/src/test/resources/org/jenkinsci/plugins/pipeline/maven/docker/NonMavenJavaContainer/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM jenkins/java:d93654cc6239
+
+RUN apt-get remove maven -y maven
```

Comment:
duplicate `maven`  no ?

Reply:
yes, you are right - i removed the second one